src/navigation/components/NavigationItem/NavigationItem.tsx

```diff
@@ -14,6 +14,13 @@ const b = block('navigation-item');
 
 const ANALYTICS_ID = 'navigation';
 
+const navigationItemTypeValues = Object.values(NavigationItemType);
+const directNavigationItemTypeValues = navigationItemTypeValues;
+directNavigationItemTypeValues.splice(
+    directNavigationItemTypeValues.indexOf(NavigationItemType.Dropdown),
+    1,
+);
+
```

Comment:
Suggest:
```suggestion
const directNavigationItemTypeValues = navigationItemTypeValues.splice(
    navigationItemTypeValues.indexOf(NavigationItemType.Dropdown),
    1,
);
```

Can you explain what is going on here with splice?

Reply:
The idea here is to get all default `NavigationItemType`s  values except `NavigationItemType.Dropdown`. Because for these values later we'll exclude some props for underlying components.